dist/focus-ring.js

```diff
@@ -244,6 +273,35 @@ function init() {
     el.removeAttribute('data-focus-ring-added');
   }
 
+  /**
+   * Validate the key we're handling should result in the application of the
+   * `focus-ring` class.
+   * @param {Event} e
+   * @return {Boolean}
+   */
+  function keyIsValid(e) {
+    var keyCode = e.keyCode;
```

Comment:
`e.keyCode` is deprecated. Use `e.code` instead.
See https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode.

Reply:
Never mind. As Rob pointed out, `KeyboardEvent.code` is not supported in Internet Explorer.
Thanks for the heads up @robdodson.